tests/integration/api_v2/rpc/export.test.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const moment = require('moment');
+
+const config = require('../../../config');
+const exportConfig = require('../../../../services/export/config');
+const {
+	request,
+} = require('../../../helpers/socketIoRpcRequest');
+const {
+	jsonRpcEnvelopeSchema,
+	invalidParamsSchema,
+	invalidRequestSchema,
+} = require('../../../schemas/rpcGenerics.schema');
+const {
+	metaSchema,
+	exportSchema,
+	exportSchemaAccepted,
+	goodRequestSchema,
+} = require('../../../schemas/api_v2/export.schema');
+
+const { waitForSuccess } = require('../../../helpers/utils');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v2`;
+const requestTransactionExport = async (params) => request(wsRpcUrl, 'get.transactions.export', params);
+const requestTransactions = async (params) => request(wsRpcUrl, 'get.transactions', params);
+describe('Export API', () => {
+	const startDate = moment('2021-01-10').format(exportConfig.csv.dateFormat);
+	const endDate = moment('2021-11-30').format(exportConfig.csv.dateFormat);
+	let refTransaction1;
+	let refTransaction2;
+	beforeAll(async () => {
+		const response1 = await requestTransactions({ limit: 1 });
+		[refTransaction1] = response1.result.data;
+
+		const response2 = await requestTransactions({ limit: 1, offset: 1 });
+		[refTransaction2] = response2.result.data;
+	});
+
+	describe('Schedule file export', () => {
+		it('Schedule file export from account address -> return 202 Accepted', async () => {
```

Comment:
Please add two more test cases:
1. Only `address` in request params
2. Only `publicKey` in request params 

Also, please update the test case description. `202 ACCEPTED`, `200 OK` are all HTTP codes.